crates/context_aware_config/src/api/config/handlers.rs

```diff
@@ -536,8 +553,97 @@ async fn reduce_config(
     Ok(HttpResponse::Ok().json(config))
 }
 
+#[cfg(feature = "high-performance-mode")]
+#[get("/fast")]
+async fn get_config_fast(
+    tenant: Tenant,
+    state: Data<AppState>,
+) -> superposition::Result<HttpResponse> {
+    use fred::interfaces::MetricsInterface;
+
+    log::debug!("Started redis fetch");
+    let config_key = format!("{}::cac_config", *tenant);
+    let last_modified_at_key = format!("{}::cac_config::last_modified_at", *tenant);
+    let audit_id_key = format!("{}::cac_config::audit_id", *tenant);
+    let config_version_key = format!("{}::cac_config::config_version", *tenant);
+    let client = state.redis.next_connected();
+    let config = client.get::<String, String>(config_key).await;
+    let metrics = client.take_latency_metrics();
+    let network_metrics = client.take_network_latency_metrics();
+    log::trace!(
+        "Network metrics for config fetch in milliseconds :: max: {}, min: {}, avg: {}; Latency metrics :: max: {}, min: {}, avg: {}",
+        network_metrics.max,
+        network_metrics.min,
+        network_metrics.avg,
+        metrics.max,
+        metrics.min,
+        metrics.avg
+    );
+    match config {
+        Ok(config) => {
+            let mut response = HttpResponse::Ok();
+            if let Ok(max_created_at) =
```

Comment:
@Datron do we need this 3 redis call for audit_id, config_version and max_created_at , we can skip for this api

Reply:
@pratikmishra356 we should maintain consistency with GET /config API, also these are not 3 API calls, they are pipelined by the redis client